tests/test_sqlalchemy/conftest.py

```diff
@@ -274,6 +274,7 @@ class UserFilter(Filter):  # type: ignore[misc, valid-type]
         age__gt: Optional[int] = None
         age__gte: Optional[int] = None
         age__in: Optional[List[int]] = None
+        age__between: Optional[List[int]] = None
```

Comment:
The type annotation here and in [examples/fastapi_filter_sqlalchemy.py:105](https://github.com/arthurio/fastapi-filter/pull/551/files/a2e98b9e2fca2e407aaa44a3c9ee6916266b7c99#diff-65d5f41a1e3fe26e9087b18857256dbf9ec71f63301ac333a54e23a0c6b7a233R105) (`List[List[int]]`) differs. Is this a mistake?